ISSUER_PROTOCOL.md

```diff
@@ -52,10 +52,14 @@ To support the redemption of tokens by the client, server paths that support tok
 
 At redemption time, the token is decoded and provided via the redemption API. The issuer can then choose to encode this in the Redemption Record in whatever way it prefers.
 
+#### Request Signing
```

Comment:
Either point a reference (as defined by [tls spec]) or include the explicit expansion (EcDSA with curve...).